lib/v2/deltaio/blog.js

```diff
@@ -0,0 +1,27 @@
+const got = require('@/utils/got');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const baseUrl = 'https://delta.io';
+    const dataUrl = `${baseUrl}/page-data/blog/page-data.json`;
+
+    const data = await ctx.cache.tryGet(dataUrl, async () => {
+        const { data } = await got(dataUrl);
+        return data;
+    });
```

Comment:
Please set the [4th parameter of tryGet](https://docs.rsshub.app/joinus/advanced/use-cache) to `false`, otherwise `data` won't receive any updates.